app/Http/Controllers/Admin/RoleController.php

```diff
@@ -72,7 +72,8 @@ public function store(Request $request): RedirectResponse
         ]);
 
         if ($request->permissions) {
-            $role->givePermissionTo($request->permissions);
+            $collectedPermissions = collect($request->permissions)->map(fn($val)=>(int)$val);
```

Comment:
Why need the map items to be casted to an int?

Reply:
https://spatie.be/docs/laravel-permission/v6/upgrading#content-upgrading-from-v5-to-v6

![Screenshot_20240726_064658_com android chrome](https://github.com/user-attachments/assets/f6892f25-8bdb-4857-8b12-a7cd2c8b0e48)

Reply:
@IceToast